config/config.default.json

```diff
@@ -25,7 +25,9 @@
         "DBInstanceEngine": "mysql",
         "DBInstanceClass": "db.r4.large",
         "DBUserName": "mmuser",
-        "DBPassword": "mostest"
+        "DBPassword": "mostest",
```

Comment:
As mentioned in a private discussion this needs to be changed to avoid DB creation failing since it requires the password length to be at least 8 chars.

Reply:
Done